src/edu/mills/cs180a/RomanNumeral.java

```diff
@@ -81,13 +84,40 @@ public String toString() {
 		return text;
 	}
 	
+
 	@VisibleForTesting
 	protected static int convertFromStringToInt(String s) {
-		return 0;
+		
+		char [] romanNums = s.toCharArray();
+
+		int numericValue = 0;
+		
+		for(char c: romanNums)
```

Comment:
I can tell the formatter isn't running. Please double-check the save actions. If you can't quickly figure out the problem, ask Ben during lab hours or me during office hours. (It's not worth your spending more than 5 minutes on.)